test/parallel/test-stream-pipeline.js

```diff
@@ -483,3 +490,30 @@ const { promisify } = require('util');
     { code: 'ERR_INVALID_CALLBACK' }
   );
 }
+
+{
+  const server = http.Server(function(req, res) {
+    res.write('asd');
+  });
+  server.listen(0, function() {
+    http.request({
```

Comment:
Nit: you could just use `http.get({ port: this.address().port })` and remove `.end()`. Less code to read.